olla/include/openmm/Platform.h

```diff
@@ -45,23 +45,42 @@ class Kernel;
 class KernelFactory;
 
 /**
- * A Platform defines an implementation of all the kernels needed to perform some calculation.
- * More precisely, a Platform object acts as a registry for a set of KernelFactory
- * objects which together implement the kernels.  The Platform class, in turn, provides a
- * static registry of all available Platform objects.
+ * A Platform defines an implementation of all the kernels needed to perform
+ * some calculation. More precisely, a Platform object acts as a registry for a
+ * set of KernelFactory objects which together implement the kernels.  The
+ * Platform class, in turn, provides a static registry of all available
+ * Platform objects.
  *
- * To get a Platform object, call
+ * OpenMM ships with platforms that provide high-performance simulation on a CPU
+ * or GPU. They are described in more detail in the User Manual. Usually, the
+ * platform is chosen automatically when a Context is constructed.
  *
+ * There are three main ways to get a Platform object:
```

Comment:
There's also another way: use `Platform::getPlatform()` to get it by index.  For example, if you want a list of all available Platforms, you would call `getNumPlatforms()` and then loop over them.